src/applications/appeals/995/components/ITFBanner.jsx

```diff
@@ -20,7 +20,7 @@ const ITFBanner = props => {
   const [messageDismissed, setMessageDismissed] = useState(false);
 
   const goHome = () => {
-    props.router.push(`${BASE_URL}/introduction`);
+    props.router.push(`/introduction`);
```

Comment:
Note: This fixes an issue that, when pressing the back button on the `/veteran-information` page, the browser tries to go back to an incorrect URL of `/decision-reviews/supplemental-claim/file-supplemental-claim-form-20-0995/decision-reviews/supplemental-claim/file-supplemental-claim-form-20-0995/introduction` (note the duplicated paths). This used to work due to the catch all redirect back to the root URL, but with the page not found it was creating issues.

Reply:
Thanks for finding and fixing this!